examples/hpa/README.md

```diff
@@ -1,3 +1,169 @@
+# custom-metrics-stackdriver-adapter example instructions
+
+The Custom Metrics Stackdriver Adapter supports querying metrics from
+Managed Service for Prometheus starting with
+[version v0.13.1 of the adapter](https://github.com/GoogleCloudPlatform/k8s-stackdriver/releases/tag/cm-sd-adapter-v0.13.1).
+
+To set up an example HPA configuration using the Custom Metrics Stackdriver
+Adapter, do the following:
+
+1. Set up Managed Service for Prometheus in your cluster.
+2. Install Custom Metrics Stackdriver Adapter in your cluster.
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml
+   ```
+
+3. Deploy an example Prometheus metrics exporter and an HPA resource:
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/custom-metrics-prometheus-sd.yaml
+   ```
+
+   This command deploys an exporter application that emits the metric `foo` and
+   an HPA resource. The HPA scales this application up to 5 replicas to achieve
+   the target value for the metric `foo`.
+
+4. Define a PodMonitoring resource by placing the following configuration in a file named `podmonitoring.yaml`.
+
+   ```yaml
+   apiVersion: monitoring.googleapis.com/v1
+   kind: PodMonitoring
+   metadata:
+     name: prom-example
+   spec:
+     selector:
+       matchLabels:
+         run: custom-metric-prometheus-sd
+     endpoints:
+     - port: 8080
+       interval: 30s
+   ```
+
+5. Deploy the new PodMonitoring resource:
+
+   ```sh
+   kubectl -n default apply -f podmonitoring.yaml
+   ```
+
+   Within a couple of minutes, {{managed_prometheus_name_short}} processes the
```

Comment:
All these {{variables}} need to be expanded out, as they're not defined here.

{{managed_prometheus_name_short}} == Managed Service for Prometheus
{{monitoring_name}} == Cloud Monitoring
{{metrics_explorer}} == Metrics Explorer

Reply:
done